### ISSUE TITLE
Add option to fill container based on item width instead

### ISSUE THREAD

**Issue by alimony** [bigbitecreative/macy.js#86]
As far as I can tell there is no way to let macy fill the container with items based on the width of each item. In my design, the width of each item is much more important than the exact number of columns at each page width. I would like an option to use whatever number of columns given a fixed column width.

**Comment by mortennh** [comment]
@alimony i have exact the same problem. the only alternative i know that supports width-based columns is desandros/masonry

https://masonry.desandro.com/options.html#columnwidth

https://codepen.io/desandro/pen/osFxj

**Comment by alimony** [comment]
Yeah, I'm actually researching alternatives to Masonry because I've found its behaviour buggy lately, maybe because it hasn't kept up with browser development. The last release was three years ago and I don't see much engagement in the issues and PRs either.

**Comment by mortennh** [comment]
@alimony i just switched two of my masonry grids from macy to desandors and it works like a charm! maybe you give it a try. i disabled the `animationDuration` since it looks shit^^ happy to help if you need assistance
